dd-sdk-android/src/main/kotlin/com/datadog/android/v2/core/DatadogCore.kt

```diff
@@ -143,6 +147,15 @@ class DatadogCore(
         webViewRumFeature?.flushStoredData()
     }
 
+    /**
+     * Returns all registered features.
+     */
+    fun getAllFeatures(): List<FeatureScope> {
+        // TODO-2138
+        // should it be a part of SDKCore?
+        return emptyList()
```

Comment:
IMO it can be

Reply:
I didn't pull it up to the `SDKCore` interface definition, because the question is why would some feature request all registered features, shouldn't we limit interface API only to getting explicit features by names? Otherwise too much power is exposed.